internal/host/validator.go

```diff
@@ -760,9 +767,10 @@ func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationConte
 		return ValidationSuccess
 	}
 
-	if len(c.host.Connectivity) == 0 {
-		return ValidationFailure
+	if len(c.host.Inventory) == 0 || len(c.host.Connectivity) == 0 {
```

Comment:
Don't depend on inventory and use host-id for the message instead

Reply:
done.